charts/spire/charts/spire-server/templates/statefulset.yaml

```diff
@@ -174,6 +213,14 @@ spec:
           configMap:
             name: {{ include "spire-controller-manager.fullname" . }}
         {{- end }}
+        {{- if eq (.Values.tornjak.enabled | toString) "true" }}
+        {{- if .Values.tornjak.config }}
+        - name: tornjak-config
```

Comment:
A lot of the object names are not templated out. They should be throughout the chart.